event_manager/Event query_target
duration target_duration